### PR TITLE
[ES] Add `Layer` and update all subarticles of `Music theory`

### PR DESCRIPTION
**Pull request by agatemosu**
## Self-check

- [x] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)
- [x] *(translations only)* The changes are reviewed on GitHub [by a fluent speaker](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#review)

### CODE REVIEW

File: wiki/Music_theory/Rhythm/es.md [hunk 1]
```diff
@@ -1,12 +1,11 @@
 ---
-no_native_review: true
 stub: true
 ---
 
 # Ritmo
 
-El **ritmo** es "el patrón de sonido, silencio y énfasis en una canción".[^rhythm-ref] Al crear [beatmaps](/wiki/Beatmap), los mappers prestan atención a dónde aterrizan los sonidos en la música para determinar dónde colocar los [objetos](/wiki/Gameplay/Hit_object) y los [hitsounds](/wiki/Beatmapping/Hitsound), con el fin de seguir el ritmo y enfatizar ciertas partes del mismo. Al ser un juego de ritmo, el ritmo es una parte intrínseca de cualquier beatmap en osu!.
+El **ritmo** es «el patrón de sonido, silencio y énfasis en una canción».[^rhythm-ref] Al crear [beatmaps](/wiki/Beatmap), los mappers prestan atención a dónde aterrizan los sonidos en la música para determinar dónde colocar los [objetos](/wiki/Gameplay/Hit_object) y los [hitsounds](/wiki/Beatmapping/Hitsound), con el fin de seguir el ritmo y enfatizar ciertas partes del mismo. Al ser un juego de ritmo, el ritmo es una parte intrínseca de cualquier beatmap en osu!.
```
Comment (UnknownOsu) [inline wiki/Music_theory/Rhythm/es.md:7]:
```suggestion
El **ritmo** es «el patrón de sonido, silencio y énfasis en una canción».[^rhythm-ref] Al crear [beatmaps](/wiki/Beatmap), los mappers prestan atención a la ubicación de los sonidos en la música para determinar dónde colocar los [objetos](/wiki/Gameplay/Hit_object) y los [hitsounds](/wiki/Beatmapping/Hitsound), con el fin de seguir el ritmo y enfatizar ciertas partes del mismo. Al ser un juego de ritmo, el ritmo es una parte intrínseca de cualquier beatmap en osu!.
```

File: wiki/Music_theory/Syncopation/es.md [hunk 2]
```diff
@@ -1,18 +1,17 @@
...
 
-Como un ejemplo simple, la [dificultad Insane de fanzhen0019 de *Chasers - Lost*](https://osu.ppy.sh/beatmapsets/151114#osu/372628) contiene patrones de [slider](/wiki/Gameplay/Hit_object/Slider) con [sliderheads](/wiki/Gameplay/Hit_object/Slider/Sliderhead) tanto en ritmo como fuera de ritmo.
+Como un ejemplo simple, la [dificultad Insane de fanzhen0019 de *Chasers - Lost*](https://osu.ppy.sh/beatmapsets/151114#osu/372628) contiene patrones de [sliders](/wiki/Gameplay/Hit_object/Slider) con [sliderheads](/wiki/Gameplay/Hit_object/Slider/Sliderhead) tanto en ritmo como fuera de ritmo.
 
-Algunos de los patrones del slider, como el primero, siguen la melodía del sintetizador que se sincopa contra las patadas de batería, aterrizando en marcas rojas de la línea de tiempo:
+Algunos de los patrones de slider, como el primero, siguen la melodía del sintetizador que se sincopa contra las patadas de batería, aterrizando en marcas rojas de la línea de tiempo:
```
Comment (UnknownOsu) [inline wiki/Music_theory/Syncopation/es.md:11]:
```suggestion
Algunos de los patrones de slider, como el primero, siguen la melodía del sintetizador que se sincopa con los golpes de la batería, aterrizando en marcas rojas de la línea de tiempo:
```

File: wiki/Music_theory/Tempo/es.md [hunk 3]
```diff
@@ -1,16 +1,36 @@
...
+::: Infobox
+![Captura de pantalla del indicador de PPM en la página de información del beatmap](img/beatmap-info.png "El tempo de la canción, igual a 187, puede verse en la esquina superior de la página de información del beatmap")
+:::
 
-El BPM de una canción en un beatmap se puede encontrar en su página de información junto al símbolo del metrónomo.
+El **tiempo** se refiere a la velocidad de una canción. Suele medirse en **pulsos por minuto** (***PPM***), es decir, el número de [pulsos musicales enteros](/wiki/Music_theory/Beat) que hay en un minuto de canción. Por ejemplo, un tempo de 60 pulsos por minuto significa un pulso por segundo, mientras que un tempo de 120 pulsos por minuto es el doble de rápido, lo que significa dos pulsos cada segundo. El tempo afecta directamente a muchos aspectos del juego, como la [velocidad del slider](/wiki/Gameplay/Hit_object/Slider/Slider_velocity), o el [kiai time](/wiki/Gameplay/Kiai_time) estroboscópico. Las páginas de información Beatmap muestran el tempo junto a un símbolo de [metrónomo](https://es.wikipedia.org/wiki/Metrónomo), un dispositivo utilizado por los músicos que ayuda a tocar con un pulso regular.
```
Comment (UnknownOsu) [inline wiki/Music_theory/Tempo/es.md:18]:
```suggestion
El **tempo** se refiere a la velocidad de una canción. Suele medirse en **pulsos por minuto** (***PPM***), es decir, el número de [pulsos musicales enteros](/wiki/Music_theory/Beat) que hay en un minuto de canción. Por ejemplo, un tempo de 60 pulsos por minuto significa un pulso por segundo, mientras que un tempo de 120 pulsos por minuto es el doble de rápido, lo que significa dos pulsos cada segundo. El tempo afecta directamente a muchos aspectos del juego, como la [velocidad del slider](/wiki/Gameplay/Hit_object/Slider/Slider_velocity), o el [kiai time](/wiki/Gameplay/Kiai_time). Las páginas de información de los beatmaps muestran el tempo junto a un símbolo de [metrónomo](https://es.wikipedia.org/wiki/Metrónomo), un dispositivo utilizado por los músicos que ayuda a tocar con un pulso regular.
```

File: wiki/Music_theory/Time_signature/es.md [hunk 4]
```diff
@@ -8,8 +7,8 @@ tags:
...
+*Para más información sobre la signatura de compás en teoría musical, véase: [Compás (música)](https://en.wikipedia.org/wiki/Time_signature)*
 
-Una **signatura de compás** es un término musical que define el número de [tiempos](/wiki/Music_theory/Beat) en un único [compás](/wiki/Music_theory/Measure) de la canción (el número superior), y qué nota indica un solo tiempo (el número de abajo). Para una signatura de compás de 4/4 (también conocido como "tiempo común"), 4 1/4 notas (*negras*) caben en un compás, mientras que una signatura de compás de 3/4 (también conocido como "vals") significa que 3 1/4 notas caben en un compás.
+Una **signatura de compás** es un término musical que define el número de [pulsoss](/wiki/Music_theory/Beat) en un único [compás](/wiki/Music_theory/Measure) de la canción (el número superior), y qué nota indica un solo tiempo (el número de abajo). Para una signatura de compás de 4/4 (también conocido como «tiempo común»), 4 1/4 notas (*negras*) caben en un compás, mientras que una signatura de compás de 3/4 (también conocido como «vals») significa que 3 1/4 notas caben en un compás.
```
Comment (UnknownOsu) [inline wiki/Music_theory/Time_signature/es.md:12]:
```suggestion
Una **signatura de compás** es un término musical que define el número de [pulsos](/wiki/Music_theory/Beat) en un único [compás](/wiki/Music_theory/Measure) de la canción (el número superior), y qué nota indica un solo tiempo (el número de abajo). Para una signatura de compás de 4/4 (también conocido como «tiempo común»), caben 4 1/4 notas (*negras*) en un compás, mientras que una signatura de compás de 3/4 (también conocido como «vals») significa que caben 3 1/4 notas en un compás.
```

File: wiki/Music_theory/Time_signature/es.md [hunk 5]
```diff
@@ -8,8 +7,8 @@ tags:
...
 
-El [editor de beatmap](/wiki/Client/Beatmap_editor) de osu! ofrece varias signatura de compás para [mapping](/wiki/Beatmapping), que se pueden seleccionar desde el [menú de Timing](/wiki/Client/Beatmap_editor/Timing). Estos incluyen 3/4, 4/4, 5/4, 6/4, y 7/4.
+El [editor de beatmaps](/wiki/Client/Beatmap_editor) de osu! ofrece varias signaturas de compás para [mapping](/wiki/Beatmapping), que se pueden seleccionar desde el [menú Timing](/wiki/Client/Beatmap_editor/Timing). Estos incluyen 3/4, 4/4, 5/4, 6/4, y 7/4.
```
Comment (UnknownOsu) [inline wiki/Music_theory/Time_signature/es.md:14]:
```suggestion
El [editor de beatmaps](/wiki/Client/Beatmap_editor) de osu! ofrece varias signaturas de compás para el [mapping](/wiki/Beatmapping), que se pueden seleccionar desde el [menú Timing](/wiki/Client/Beatmap_editor/Timing). Estos incluyen 3/4, 4/4, 5/4, 6/4, y 7/4.
```

**Review by UnknownOsu — CHANGES_REQUESTED** [review]

File: wiki/Music_theory/Time_signature/es.md [hunk 6]
```diff
@@ -8,8 +7,8 @@ tags:
 
 # Signatura de compás
 
-*Para más información sobre la signatura de compás en teoría musical, véase: [Time signature (en inglés)](https://en.wikipedia.org/wiki/Time_signature)*
+*Para más información sobre la signatura de compás en teoría musical, véase: [Compás (música)](https://en.wikipedia.org/wiki/Time_signature)*
```
Comment (UnknownOsu) [inline wiki/Music_theory/Time_signature/es.md:10]:
```suggestion
*Para más información sobre la signatura de compás en teoría musical, véase: [Compás (música)](https://es.wikipedia.org/wiki/Compás_(música))*
```

**Review by UnknownOsu — CHANGES_REQUESTED** [review]

File: wiki/Music_theory/Measure/es.md [hunk 7]
```diff
@@ -8,8 +7,8 @@ tags:
 
 # Compás
 
-![Un compás en la línea de tiempo de la pestaña de timing](img/measure_timing.png "Las marcas blancas representan tiempos, y las más grandes representan el comienzo de un nuevo compás.")
+![Un compás en la línea de tiempo de la pestaña de timing](img/measure_timing.png "Las marcas blancas representan pulsos, y las más grandes representan el comienzo de un nuevo compás.")
 
-Un **compás** en teoría musical se refiere a una sola unidad de tiempo que incluye un número específico de [tiempos](/wiki/Music_theory/Beat) (determinado por la [signatura de compás](/wiki/Music_theory/Time_signature)) tocado a un [tempo](/wiki/Music_theory/Tempo) particular.
+Un **compás** en teoría musical se refiere a una sola unidad de tiempo que incluye un número específico de [pulsos](/wiki/Music_theory/Beat) (determinado por la [signatura de compás](/wiki/Music_theory/Time_signature)) tocado a un [tempo](/wiki/Music_theory/Tempo) particular.
```
[the remark this answers is not in the document]
Comment (agatemosu) [reply wiki/Music_theory/Measure/es.md:12]: nota: en femenino

**Review by TicClick — APPROVED** [review]

**Pull request merged**